cwms_radar_api/src/main/java/cwms/radar/data/dao/OfficeDao.java

```diff
@@ -74,7 +74,7 @@ public Optional<Office> getOfficeById(String officeId) {
                 view.OFFICE_TYPE.as("type"),
                 view.REPORT_TO_OFFICE_ID.as("reportsTo"))
             .from(view)
-            .where(view.OFFICE_ID.eq(officeId))
+            .where(view.OFFICE_ID.eq(officeId.toUpperCase()))
```

Comment:
Is there a reason we shouldn't be doing this? Confirmed swagger returns an error and claims the office id is not present. Could also set the case in JOOQ _vs_ java toUpperCase

Reply:
Yeah, that should have the JOOQ too upper on both. at the least both *SHOULD* be obviously marked at an upper case comparison.